tests/integration/long/test_loadbalancingpolicies.py

```diff
@@ -520,6 +520,89 @@ def test_token_aware_with_local_table(self):
 
         cluster.shutdown()
 
+    def test_token_aware_with_shuffle_rf2(self):
+        """
+        Test to validate the hosts are shuffled when the `shuffle_replicas` is truthy
```

Comment:
You're right, is not that straightforward to check randomness is happening. I updated the tests following your suggestion.